README.md

```diff
@@ -2,12 +2,47 @@
 
 ---
 
-> TechStack - React JS, Material UI
+## Installation
 
-_MORE INFO YET TO BE ADDED... STAY TUNED_
+The Open Event Frontend can be easily deployed on a variety of platforms, so just run the following commands when setting up for the first time:
 
-# DON'T JUST CLONE, PLEASE LEAVE A STAR, MOTIVATES US TO ADD MORE FEATURES TO THE PROJECT.
+- `yarn install or npm install  `
+- `yarn start or npm start `
+- Visit your app at [http://localhost:3000](http://localhost:3000).
 
-> NEW UI COMING UP , CHECK THIS SITE TO SEE THE LATEST UPDATES : [https://cowinvaccinetracker.stephinreji.me/](https://cowinvaccinetracker.stephinreji.me/)
+CoWIN Vaccination Slots Checking App is a website, mobile-ready,
+REACT JS-along with Material-UI
 
-### _NOTE:_ Please check the Projects section to see the progress and the discussion panel to add your ideas, which will eventually be picked up for development
+## Features
+
+CoWIN Vaccination Slots Checking App is a user Friendly website
+that allow users to find vaccine in nearby available Center.
+
+> The overriding design goal for CoWIN is
+> helping people around nation to
+> book their slot by providing details of
```

Comment:
We don't help in booking slots, we can only view the slots.

Please change the text here